lib/zenodo/dsl/deposition_actions.rb

```diff
@@ -8,8 +8,8 @@ module DSL::DepositionActions
     # @param [String, Fixnum] id A deposition's ID.
```

Comment:
Please update the comments to reflect the param change. https://gist.github.com/chetan/1827484#methods

```
# @param [Hash] options The options to publish a deposition with.
# @option options [String, Fixnum] :id A deposition's ID.
```